widget/check_group.go

```diff
@@ -44,6 +45,24 @@ func (r *CheckGroup) Append(option string) {
 	r.Refresh()
 }
 
+// Remove removes the first specified option found from a CheckGroup widget.
+//
+// Since: 2.3
+func (r *CheckGroup) Remove(s string) bool {
+	found := -1
+	for i, o := range r.Options {
+		if strings.EqualFold(s, o) {
+			found = i
+			continue
+		}
+	}
+	if found == -1 {
+		return false
+	}
+	r.Options = append(r.Options[:found], r.Options[found+1:]...)
+	return true
+}
```

Comment:
Would it not be better to do something like this?
```go
	for i, o := range r.Options {
		if strings.EqualFold(s, o) {
			r.Options = append(r.Options[:i], r.Options[i+1:]...)
			return true
		}
	}
	return false
}
```

Reply:
Possible, but even in my version I am afraid of the moving states with no lock between append and remove. It is even worse to change states in a loop but that is your call :) I'll made the changes.